### PR TITLE
Migrate sense uploader to use backend listener

### PR DESCRIPTION
**Pull request by artem-fedorov**

### CODE REVIEW

File: plugins/sense-uploader/pom.xml [hunk 1]
```diff
@@ -59,7 +59,6 @@
             <groupId>kg.apc</groupId>
             <artifactId>jmeter-plugins-emulators</artifactId>
             <version>0.2</version>
-            <scope>test</scope>
```
Comment (undera) [inline plugins/sense-uploader/pom.xml]: Why this change? Emulators are not supposed to be used for anything except testing.

Comment (artem-fedorov) [inline plugins/sense-uploader/pom.xml]: Replaced by `components` dependency

**Comment by undera** [comment]
Did you test it well? And especially from non-gui environment, like Jenkins?

**Comment by artem-fedorov** [comment]
Yes, I tested it on Jenkins, and then compare results at Sense and Jenkins Performance Plugin. They're equals.
